TODO.md

```
@@ -0,0 +1,266 @@
+# Spec Test Anomalies
+
+Issues discovered while implementing portable test specs from `specification/md/test/specs/`.
+
+Related: [specification PR #421](https://github.com/ably/specification/pull/421) - Portable unit test specifications
+
+## Test Results Summary (Latest Run)
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)